postprocessing/OEP_MaStR_OPSD_Comparison.ipynb

```diff
@@ -1656,9 +1656,7 @@
     "import os\n",
     "import geopandas as gpd\n",
     "from shapely.geometry import Point\n",
-    "import upsetplot\n",
-    "\n",
-    "from helpers import *"
```

Comment:
`helpers` was not used here (and bokeh returns a 1500 line output in the notebook...)